src/kremling.d.ts

```diff
@@ -8,6 +8,7 @@ declare namespace Kremling {
   function m(className: string, condition: any): KremlingString & string;
   function toggle(truthyClass: string, falsyClass: string, condition: any): KremlingString & string;
   function t(truthyClass: string, falsyClass: string, condition: any): KremlingString & string;
+  function k(strings: Array<string>, ...args: Array<string>): object | string;
```

Comment:
Might be good to define the object as as KremlingLoaderParse object or something. And then change the `useCss` type def to also allow for that to be passed in.

If you're not a typescript user, though, don't worry about it. I use typescript a bit now so I would probably get around to changing to when I see it not work in my project

Reply:
I'll let you do that 😄